libs/core/src/components/card/card.ts

```diff
@@ -6,35 +6,38 @@ import {
   gdsCustomElement,
   html,
 } from '../../utils/helpers/custom-element-scoping'
-import { GdsContainer } from '../container'
+import { GdsDiv } from '../div'
 
 /**
  *  A container for content representing a single entity. e.g. a contact, article, or task.
  *
- * `gds-card` inherits all properties from `gds-container`, but comes with some predefined defaults
+ * `gds-card` inherits all properties from `gds-div`, but comes with some predefined defaults
  * and a shortcut property for specifiying different card variants.
  *
  * @element gds-card
  * @status beta
  *
  */
 @gdsCustomElement('gds-card')
-export class GdsCard extends GdsContainer {
+export class GdsCard extends GdsDiv {
   static styles = [
     tokens,
     css`
       :host {
+        box-sizing: border-box;
         border-color: var(--_border-color);
         background-color: var(--_background-color);
         color: var(--_color);
+        border-width: var(--gds-space-4xs);
+        border-style: solid;
       }
     `,
   ]
 
   /**
-   * Controls the box-shadow property.
+   * Style Expression Property for the `box-shadow` property.
    *
-   * These are the available values you can use to define shadow size
+   * Accepts shadow tokens from the design system.
    *
    * `xs`, `s`, `m`, `l`, `xl`
```

Comment:
The shadows seem to not be working!

Reply:
The shadow property is still only available on `gds-card`.

Originally we said that shadows should only ever be used on cards, but perhaps it would be best to move that one up to `gds-div` as well, for consistency. What do you think?

Reply:
Ok, here's what I did:
- Add `box-shadow` to `gds-div`, to keep in line with "same names as css"
- Keep `shadow` on `gds-card` for now, but tag as deprecated